src/js/setUpSite.js

```diff
@@ -203,6 +205,13 @@ const setUpCitiesLayer = async (map) => {
     setMapToCity(map, cityId, cities[cityId]);
   });
 
+  // Set up map to update when user clicks within a city's boundary
+  allBoundaries.addEventListener("click", (e) => {
+    const cityId = e.sourceTarget.feature.properties.id;
+    cityToggleElement.value = cityId;
+    setMapToCity(map, cityId, cities[cityId]);
```

Comment:
Ik this is still a draft so pardon if this is premature. We'll still want to only do this if the zoom is a certain value. Otherwise clicking on the city from zoomed out will be noisy.

Reply:
When you click a city, it zooms in onto the city (similar to if you select it from the dropdown). I think this resolves the noise you're talking about?

Reply:
Not quite. Imagine you're super zoomed out on the map, so the cities are tiny dots. You click on it because let's say you're on a phone and the touch targets aren't big—but really you were just trying to zoom in or move around the map!

We should only change the city if the zoom level is below a certain threshold.